### ISSUE TITLE
Internal Error: database.lua:111: No such tableName table

### ISSUE THREAD

**Issue by jbmorley** [inseven/opolua#347]
## Description

https://archive.org/download/3-libjune-05/3LIBJUNE05.iso/library%2Fepocmisc%2Fbeattest%2Fbeattest.sis

## Details

```
database.lua:111: No such tableName table
BEATTEST\BEATTEST:000004AA: B4 [Open] logName=0 fields=G%, I, J%, A%
TOOLBAR\TBARLINK: [Lua] modules/toolbar.lua:68
BEATTEST\INIT:00000176: 53 [RunProcedure] 0x0067 (name="TBARLINK" nargs=1)
stack traceback:
	[C]: in function 'assert'
	database.lua:111: in method 'setView'
	runtime.lua:642: in method 'openDb'
	ops.lua:1604: in function 'ops.Open'
	runtime.lua:864: in function <runtime.lua:849>
	[C]: in function 'xpcall'
	runtime.lua:983: in method 'pcallProc'
	runtime.lua:1029: in method 'callProc'
	modules/toolbar.lua:68: in field 'fn'
	runtime.lua:310: in method 'pushNewFrame'
	ops.lua:998: in function 'ops.RunProcedure'
	runtime.lua:864: in function <runtime.lua:849>
	[C]: in function 'xpcall'
	runtime.lua:983: in method 'pcallProc'
	runtime.lua:1253: in function 'runtime.runOpo'
```

**Comment by tomsci** [comment]
Already fixed in The Great Database Refactor™ Coming Soon to a branch near you

**Comment by jbmorley** [comment]
Better than 'The Great Database Disaster...'.

**Comment by tomsci** [comment]
fixed in #350 (b267e3e03b14b3f2cf77fa673936c1030ecdb056)
